ParisBNF/abb/BNFabb134.xml

```diff
@@ -0,0 +1,452 @@
+                              </layout>
+                           </layoutDesc>
+
+                          </objectDesc>
+                        <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                              <seg type="script">Late 13th/mid-14th century.</seg>
+                              <seg type="ink">Black; red.</seg>
+                              <desc>Careful, slightly irregular; by a good scribe.
+                                In their catalogues,
+                                Conti Rossini and Chaîne dated the manuscript to
+                                the <date>15th</date> and <date>14th/15th century</date>, resp., on the palaeographical grounds.
+                                Uhlig suggested the time around the middle or shortly before the end of the 14th century
+                                (but pertaining to the earliest palaeographic Period I).
+                                However, somewhat earlier dating can still be justified, basing on the palaeographic
+                                traits and orthography (the word for God consistently written in two words, እግዚአ፡ ብሔር፡).
+
+
+                                </desc>
+                              </handNote>
+
+
+                        </handDesc>
+
+                            <additions>
+                              <list>
+
+                                      <item xml:id="a1">
+                                         <locus target="#34r"/>
+                                         <desc type="OwnershipNote"/>
+                                         <q xml:lang="gez">
+                                           <q xml:lang="gez">ዝመጽሐፍ፡ ዘ<persName ref="PRS1071dAbbadi" role="owner">እንጦንዮስ፡ ዘአበዩደ፡</persName>
+                                            ዘአጥረዮ፡ በንዋዩ፡</q>
+                                          </q>
+                                      </item>
+
+                                      <item xml:id="a2">
+                                         <locus target="#51va"/>
+                                         <desc type="Unclear">A note, written in a much later hand than the main <ref target="#h1"/>,
+                                           that seems to be ascribing the authorship of <ref target="ms_i3"/>
+                                           to <persName ref="PRS5720JohnChr">John Chrysostom</persName></desc>
+                                           <q xml:lang="gez">
+                                           ዘዮሐንስ፡ ኢጲስቆጶስ፡ <sic resp="DN"></sic>ዘኵስጥንጥንያ፡</q>
+                                         </item>
+
+                                         <item xml:id="a4">
+                                            <locus target="#65ra"/>
+                                            <desc type="Inventory">A short inventory of works or manuscripts,
+                                              crudely written in a recent hand</desc>
```

Comment:
Can the language be discerned, here and in the additions below?

Reply:
Well, yes, to a certain degree.